### PR TITLE
fix: Allow lambda inside VPC

### PR DESCRIPTION
**Pull request by jsalcedo1987**
Hi! 👋

This PR contains the following changes:
-  It allows webhook lambda to run inside a VPC 
-  fix a minor issue in the syncer module, when the lambda is running inside vpc AWSLambdaVPCAccessExecutionRole needs to be attached.

Updated doc.

Do not hesitate to ask if you need further details,
Thanks!

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -161,6 +161,10 @@ module "webhook" {
   role_permissions_boundary = var.role_permissions_boundary
   repository_white_list     = var.repository_white_list
 
+  lambda_subnet_ids         = var.lambda_subnet_ids
```
Comment (npalm) [inline main.tf:164]: Can you also update the file `modules/multi-runner/webhook.tf` and pass the new varabiles down to the webhook module.

**Review by npalm — CHANGES_REQUESTED** [review]
@jsalcedo1987 Thanks for your contribution. A small remark. We basides the top level main module also the variables needs to be added to the mult-runner module.

File: main.tf (hunk 1 above)
Comment (jsalcedo1987) [reply main.tf:164]: Thanks for the heads up! 😅 I'm not using the new multi-runner module so I missed that bit. I've now added the missing variables to the file 👍

**Review by npalm — APPROVED** [review]
@jsalcedo1987 thanks looks all good now

**Pull request merged**